resources/assets/js/components/UpcomingColumn.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+
+export default class UpcomingColumn extends Component {
+  render() {
+    return(
+      this.props.events.length > 0 ?  
+      <div className="column">
+      <div className="column-title">{
+        this.props.start.getDate() == this.props.today.getDate() ? "Today" : 
+          (this.props.start.getDate() == this.props.today.getDate() + 1 ? "Tomorrow" : 
+              this.props.start.getMonth() + 1 + "/" + this.props.start.getDate())
+      }
+      </div>
+      <div className="column-content"> {
+        this.props.events.map((e, i) => 
```

Comment:
Use `(e, _)` if `i` is not used

Reply:
Ok. Thanks for your suggestion!

Just finished! Great idea, Jason.